src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -367,6 +367,16 @@ ov::Tensor ov::npuw::util::to_f16(const ov::Tensor& t) {
     return ov::npuw::util::XARCH::to_f16(t);
 }
 
+ov::Tensor ov::npuw::util::drop_last_dims(const ov::Tensor& t) {
+    const auto& shape = t.get_shape();
+    NPUW_ASSERT(shape.size() == 4);
+    const auto& strides = t.get_strides();
+    return ov::Tensor(t.get_element_type(),
+                      ov::Shape{shape[0], shape[1]},
+                      t.data(),
+                      ov::Strides{strides[0], strides[1]});
```

Comment:
There's a risk of having a Tensor over a dangling pointer, if the result of the previous operation (in the LT unroll pipeline) isn't stored anywhere.